db/migrate/20220219191425_create_user.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.string :Bio
+      t.date :UpdatedAt
+      t.date :CreatedAt
+      t.integer :PostsCounter
```

Comment:
- [ ] When naming table columns, I would suggest they start with small letters. for example, instead of `t.string :Name` use `t.string :name`. Kindly change this for all the columns in all the tables. This is a rails convetion.

Reply:
ok I'll do that.